webapp/tests/forms/flows/test_lotse_flow.py

```diff
@@ -151,6 +149,9 @@ def test_if_unkown_field_class_given_then_raise_value_error(self):
 
 
 class TestLotseInit(unittest.TestCase):
+    @pytest.fixture(autouse=True)
+    def attach_fixtures(self, test_request_context):
+        self.req = test_request_context
```

Comment:
We have to set the test_request_context to self to keep it around for all the tests, correct?

Reply:
Ah, that's something I'd been wondering about how to do… In the cases where the tests don't use `self.req`, it's mainly for documentation purposes. The way pytest fixtures work, it's enough to have use fixture function (`attach_fixtures` in this case) that takes a parameter whose name corresponds to that of an existing fixture and the fixture will be used. Setting it on self is only really needed if we want to access it in tests.